### PR TITLE
fix(uno.sdk): ignore xaml files in Platforms folders

### LINKED ISSUE #18538: XAML files in the `Platforms` folder are not filtered
### Current behavior

`.xaml` files placed in the `Platforms` folder are not filtered per platform, while `.cs` are properly included in their respective platforms.

### Expected behavior

All files placed in the `Platforms` folder are filtered.

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by ramezgerges**
GitHub Issue (If applicable): closes #18538

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.Sdk/targets/Uno.Common.targets [hunk 1]
```diff
@@ -115,6 +115,7 @@
 		<_IgnorePlatformFiles Include="$(WindowsProjectFolder)**"
 			Condition=" $(IsWinAppSdk) != 'true' AND '$(WindowsProjectFolder)'!='' " />
 		<Compile Remove="@(_IgnorePlatformFiles)" />
+		<Page Remove="@(_IgnorePlatformFiles)" />
```
Comment (ramezgerges) [inline src/Uno.Sdk/targets/Uno.Common.targets:118]: @jeromelaban Why are these "remove files from other platforms" block repeated a  million times? Also, they're not identical (e.g. this version has a `None Incude=".."` while others `Exclude` instead.

Comment (jeromelaban) [reply src/Uno.Sdk/targets/Uno.Common.targets:118]: I'm not sure to understand the question, where do you see too many repetitions?

Comment (ramezgerges) [reply src/Uno.Sdk/targets/Uno.Common.targets:118]: I'm talking about the ones in this PR (just 4 times, not a million :)). It seems like one one of these should be enough. Maybe the one in Uno.Common.targets.

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.5

**Pull request merged**